src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -130,6 +130,16 @@ class TezosNodeOperator(
       .runAsyncGetQuery(network, s"blocks/${blockHash.value}/context/contracts/${accountId.id}")
       .map(fromJson[Account])
 
+  /**
+    * Given an account's address, get it's balance/stake.
```

Comment:
typo: it's -> its

Reply:
The doc comments are no longer up-to-date with the method signature

Reply:
you're right, changed